demography.md

```diff
@@ -29,34 +29,56 @@ from IPython.display import SVG
 
 ## Population structure
 
+All population structure and demography is passed to {func}`msprime.sim_ancestry` via a {class}`msprime.Demography` object.
+
+```{code-cell} ipython3
+dem = msprime.Demography()
+```
+
 ``msprime`` supports simulation from multiple discrete populations,
-each of which is initialized with a {class}`msprime.Population` object.
+each of which is initialized in {class}`msprime.Demography` via the {meth}`msprime.Demography.add_population` method.
 For each population, you can specify a sample size, an effective population size
 at time = 0, an exponential growth rate and a name.
+In this default setup, the samples will be drawn from time = 0, but see [here](https://tskit.dev/msprime/docs/latest/ancestry.html#sampling-time) if you wish to change this.
 
 *Note: Population structure in ``msprime`` closely follows the model used in the
 ``ms`` simulator.
 Unlike ``ms`` however, all times and rates are specified
 in generations and all populations sizes are absolute (that is, not
-multiples of :math:`N_e`).*
+multiples of ``population_size``).*
 
 Suppose we wanted to simulate three sequences each from two populations
 with a constant effective population size of 500.
 
 ```{code-cell} ipython3
-pop0 = msprime.Population(initial_size=500, growth_rate=0, name='Population0')
-pop1 = msprime.Population(initial_size=500, growth_rate=0, name='Population1')
-samples0 = msprime.SampleSet(num_samples=3, population='Population0')
-samples1 = msprime.SampleSet(num_samples=3, population='Population1')
+dem = msprime.Demography()
+dem.add_population(
+  name="Population0",
+  description="Plotted in red.",
+  initial_size=500,
+  growth_rate=0
```

Comment:
Maybe omit the growth rate initially? It defaults to zero, and will perhaps distract people from the bits we're explaining at the start. 

Another thing is that these notebooks are quite "wide" when rendered in the browser. It's often better visually to put parameters on the same line (so long as they aren't cut off horizontally when rendered)